apps/docs/content/components/Hero/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Hero
+description: Use the hero component to render a full-width banner on a webpage
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image displaying a hero with a heading, description, primary and secondary actions.](https://github.com/primer/brand/assets/912236/51f3dd23-9ac7-4825-9169-4fb7f295e7f1)
+
+- Heading: the main title of the hero.
+- Description: a short description of the hero.
+- Actions: the main and secondary actions the user should take on the page.
+
+## Usage
+
+Use the hero component to provide a title, description and actions to introduce the main content of a page. The hero component is designed to be positioned at the top of a page and spans the full width of the viewport.
+
+Both heading and description of the hero should be short and descriptive. Heading should be not be longer than 2 lines, and description should be no longer than 3-4 lines.
+
+The hero component is not designed to be used as a banner or advertisement, or to provide a lot of information to the user. Consider using other components, such as the [section intro](/components/SectionIntro) or the [river](/components/River) instead.
```

Comment:
> used as a banner

Could we be more specific about this? Only because the description in the front-matter describes this component as a `full-width banner`, so clarity may be helpful on which type of banner we don't want to qualify this as.

Reply:
Also worth adding CTA Banner into the mix?

```suggestion
The hero component is not designed to be used as a banner or advertisement, or to provide a lot of information to the user. Consider using other components, such as the [CTA banner](/components/CTABanner), [section intro](/components/SectionIntro) or the [river](/components/River) instead.
```

Reply:
Agree!

Reply:
The `CTA banner` is used in most cases to repeat the content and actions the `Hero` introduces.

For this, I'd extend the first paragraph and suggest the `CTA Banner` as an optional component when not positioned in the top.